cmd/utils.go

```diff
@@ -136,16 +137,28 @@ func GetFileDetails(filePath string) (modFileContent []byte, modFileStat os.File
 	return
 }
 
-func outputToMap(output string) map[string]bool {
-	lineOutput := strings.Split(output, "\n")
-	var result []string
+func outputToMap(output string, errorOutput string) map[string]bool {
 	mapOfDeps := map[string]bool{}
+
+	// Parse dependency graph output
+	lineOutput := strings.Split(output, "\n")
 	for _, line := range lineOutput {
 		splitLine := strings.Split(line, " ")
 		if len(splitLine) == 2 {
 			mapOfDeps[splitLine[1]] = true
-			result = append(result, splitLine[1])
 		}
 	}
+
+	// Parse dependency resolution output, sent to the error output by go mod graph
+	lineOutput = strings.Split(errorOutput, "\n")
+	for _, line := range lineOutput {
+		if strings.HasPrefix(line, "go: finding") {
+			lineContent := line[12:]
```

Comment:
To make ```lineContent := line[12:]``` easier to understand, I suggest making the following change.
```
	prefix := "go: finding"
	index := len(prefix) + 1
	for _, line := range lineOutput {
		if strings.HasPrefix(line, prefix) {
			lineContent := line[index:]
```

Reply:
Done!